src/strophe.stream-management.js

```diff
@@ -164,7 +224,29 @@ Strophe.addConnectionPlugin('streamManagement', {
 			}
 
 			this._requestHandler = this._c.addHandler(this._handleServerRequestHandler.bind(this), this._NS, 'r');
+			this._ackHandler = this._c.addHandler(this._handleServerAck.bind(this), this._NS, 'a');
 			this._incomingHandler = this._c.addHandler(this._incomingStanzaHandler.bind(this));
+
+			// FIXME handler instances stored, but never used
```

Comment:
Does this result in leaked handlers or handlers that would get called when they shouldn't due to a successful reconnect?